app/views/members/autocomplete_for_member.json.erb

```diff
@@ -27,16 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
+<%#
+  Lists found users (principals) to be used for the select box.
+  If the user enters an email address, a 'result' in the form of
+  'Invite: <email>' will be returned.
+#%>
+
 {
   "results":
   {
+    <%# Offer to invite via email unless it's one of the found user's addresses already. #%>
+    <% email = params[:q] =~ /.+@.+/ && @principals.none? { |p| p.mail == params[:q] } %>
```

Comment:
The regex probably works due to they way we restrict our logins, but I'd like to see some more explicit regex here for.

Something like `/\A\S+@\S+\.\S+\z/` is vague enough that we don't create false negatives, but  at least I cannot create an invite without TLD to `deine@muddi`